build.mill.scala

```diff
@@ -106,44 +122,46 @@ object `scala3-graal-processor` extends Cross[Scala3GraalProcessor](Scala.scala3
     with CrossScalaDefaultToInternal
 
 object `scala-cli-bsp` extends JavaModule with ScalaCliPublishModule {
-  def ivyDeps = super.ivyDeps() ++ Seq(
+  def ivyDeps: Target[Agg[Dep]] = super.ivyDeps() ++ Seq(
     Deps.bsp4j
   )
-  def javacOptions = T {
+  def javacOptions: Target[Seq[String]] = Task {
     super.javacOptions() ++ Seq("-target", "8", "-source", "8")
   }
 }
 object integration extends CliIntegration {
   object test extends IntegrationScalaTests {
-    def ivyDeps = super.ivyDeps() ++ Seq(
+    def ivyDeps: Target[Loose.Agg[Dep]] = super.ivyDeps() ++ Seq(
       Deps.jgit,
       Deps.jsoup
     )
   }
   object docker extends CliIntegrationDocker {
     object test extends ScalaCliTests {
-      def sources = T.sources {
+      def sources: Target[Seq[PathRef]] = Task.Sources {
         super.sources() ++ integration.sources()
       }
-      def tmpDirBase = T.persistent {
-        PathRef(T.dest / "working-dir")
+      def tmpDirBase: Target[PathRef] = Task(persistent = true) {
+        PathRef(Task.dest / "working-dir")
       }
-      def forkEnv = super.forkEnv() ++ Seq(
+      def forkEnv: Target[Map[String, String]] = super.forkEnv() ++ Seq(
         "SCALA_CLI_TMP"                -> tmpDirBase().path.toString,
         "SCALA_CLI_IMAGE"              -> "scala-cli",
         "SCALA_CLI_PRINT_STACK_TRACES" -> "1"
       )
     }
   }
+  @unused
```

Comment:
why do we need the unused here? It's not private os should never really be marked as such